src/views/FontWatcher.tsx

```diff
@@ -0,0 +1,61 @@
+import { reaction } from "mobx";
+import { observer } from "mobx-react-lite";
+import React from "react";
+import { Preferences } from "../hooks/stores/preferences";
+
+interface Props {
+  preferences: Preferences;
+}
+
+const DEFAULT_FONTS = {
+  heading: '"Hubot Sans", "IBM Plex Mono", sans-serif',
+  body: '"Mona Sans", sans-serif',
+  mono: '"IBM Plex Mono", monospace',
+};
+
+export const FontWatcher: React.FC<Props> = observer(({ preferences }) => {
+  React.useEffect(() => {
+    return reaction(
+      () => preferences.fonts,
+      (fonts) => {
+        const root = document.documentElement;
+
+        // Apply font preferences to CSS variables
+        if (fonts?.heading) {
+          root.style.setProperty("--font-heading", fonts.heading);
+        } else {
```

Comment:
We should not have else behavior here. If the font is not defined, we should fallback to the base inheritance and, failing that, simply fallback to whatever the default stylesheet provides.

For instance if --font-heading is not defined, then it being undefined (or whatever null value is idiomatic for css variables), that should be ok so long as it then falls back to the base style (e.g. something like serif, sans-serif, or mono). If the user doesn't like it, they can easily go back and set a real value for that variable. 

In short, find a way to remove these else statements on each of the variables.